includes/abstracts/abstract.llms.admin.table.php

```diff
@@ -434,11 +456,7 @@ public function get_table_filters_html() {
 	 */
 	public function get_table_html() {
 
-		$classes = array(
-			'llms-table',
-			'llms-gb-table',
-			'llms-gb-table-' . $this->id,
-		);
+		$classes = $this->get_table_classes();
 
 		if ( $this->is_zebra ) {
```

Comment:
@pondermatic I think these conditionally-added classes should probably get moved into the new `get_table_classes()` method so that they can be modified by the new filter in that method. Do you agree?

Reply:
I don't understand. The classes **were** moved from `get_table_html()` to `get_table_classes()`.

Reply:
Pointer here was unclear, sorry. I'm referring to the classes that haven't been moved on lines 461 - 467 of the file after your new method is added.

```php
if ( $this->is_zebra ) {
   $classes[] = 'zebra`; 
}
etc..
```

Reply:
I'm glad you caught that. Can't believe I missed it!